pom.xml

```diff
@@ -705,6 +705,12 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
```

Comment:
What's the different with `-Dmaven.javadoc.skip=true`?

Reply:
-Dmaven.javadoc.skip=true on windows will be fail

Reply:
> -Dmaven.javadoc.skip=true on windows will be fail

Why? Could you check it again? I think it doesn't make sense.

Reply:
Thank you for your advice,I found a solution to the problem,Please trigger GitHub workflow again